docs/docs/configuration/db.md

```diff
@@ -53,12 +53,15 @@ $ trivy image --download-db-only
 ```
 $ trivy image --db-repository registry.gitlab.com/gitlab-org/security-products/dependencies/trivy-db
 ```
-
+    
 !!!note
     Trivy automatically adds the `trivy-db` schema version as a tag if the tag is not used:
 
     `trivy-db-registry:latest` => `trivy-db-registry:latest`, but `trivy-db-registry` => `trivy-db-registry:2`.
 
+!!!note
+    Trivy expects the OCI Artifact to have the media type `application/vnd.aquasec.trivy.db.layer.v1.tar+gzip`
```

Comment:
We also want to document the media type for Java DB.
https://github.com/aquasecurity/trivy-java-db/blob/349526b921f8709158affca196db4f3f5be36883/.github/workflows/cron.yml#L55